src/database/migrations/20210907231148_create_table_medico.js

```diff
@@ -0,0 +1,15 @@
+exports.up = async function(knex) {
+    return await knex.schema.createTable('medico', table => {
+        table.increments('id').primary();
+        table.integer('id_usuario').references('usuario.id').notNullable().onDelete('CASCADE');
+        table.text('crm').notNullable();
```

Comment:
Seria bom colocar o atributo crm com uma unique.
```suggestion
        table.text('crm').unique().notNullable();
```